packages/web3torrent/config/webpack.config.js

```diff
@@ -547,7 +550,12 @@ module.exports = function(webpackEnv) {
       // It is absolutely essential that NODE_ENV is set to production
       // during a production build.
       // Otherwise React will be compiled in the very slow development mode.
-      new webpack.DefinePlugin(stringifiedEnv),
+      new webpack.DefinePlugin({
+        ...stringifiedEnv,
+        VERSION: JSON.stringify(gitRevisionPlugin.version()),
+        COMMIT_HASH: JSON.stringify(gitRevisionPlugin.commithash()),
+        BRANCH: JSON.stringify(gitRevisionPlugin.branch())
+      }),
```

Comment:
When I tried this out, I did not see these new variables from the output of `console.log(process.env)`
![Screenshot 2020-05-07 at 10 13 18](https://user-images.githubusercontent.com/1833419/81277945-f88b1480-904c-11ea-93e4-f6d6db1a89ad.png)

Reply:
Oops, I did a lot of rebases and broke this. Should be fixed now.

Reply:
Hmm I still have this problem when checking out this branch. Does it work for you?

Reply:
Hang on I didn't pull your latest changes...